plotly/figure_factory/_county_choropleth.py

```diff
@@ -563,11 +563,15 @@ def create_choropleth(fips, values, scope=['usa'], binning_endpoints=None,
     if not gp or not shapefile or not shapely:
         raise ImportError(
             "geopandas, pyshp and shapely must be installed for this figure "
-            "factory.\n\nRun the following commands in the terminal to "
-            "ensure that the correct versions of the modules are installed:\n"
+            "factory.\n\nIf you are using OS X run the following commands in "
```

Comment:
this is os x and linux right?

Maybe this could be streamlined by saying:
Run the following commands to install the correct versions of the following modules:

If you are using Windows, follow this post to properly install geopandas and dependencies: http://geoffboeing.com/2014/09/using-geopandas-windows/

Reply:
I like the wording: the set of operating systems are fully covered.